SCClassLibrary/Common/Core/Object.sc

```diff
@@ -169,6 +179,13 @@ Object  {
 	valueEnvir { ^this }
 	valueArrayEnvir { ^this }
 
+	valueWith {|argumentsArray=([]), keywordArgumentEnvir=(())|
```

Comment:
default arguments are given in `makePerformableArray` so `argumentsArray=([])` can simply be `argumentsArray`. This will be more efficient. Same for `keywordArgumentEnvir`.